src/ci/docker/disabled/aarch64-gnu/Dockerfile

```diff
@@ -0,0 +1,85 @@
+FROM ubuntu:16.04
+
+RUN apt-get update -y && apt-get install -y --no-install-recommends \
+      bc \
+      bzip2 \
+      ca-certificates \
+      cmake \
+      cpio \
+      curl \
+      file \
+      g++ \
+      gcc-aarch64-linux-gnu \
+      git \
+      libc6-dev \
+      libc6-dev-arm64-cross \
+      make \
+      python2.7 \
+      qemu-system-aarch64 \
+      xz-utils
+
+ENV ARCH=arm64 \
+    CROSS_COMPILE=aarch64-linux-gnu-
+
+WORKDIR /build
+
+# Compile the kernel that we're going to run and be emulating with. This is
+# basically just done to be compatible with the QEMU target that we're going
+# to be using when running tests. If any other kernel works or if any
+# other QEMU target works with some other stock kernel, we can use that too!
+#
+# The `config` config file was a previously generated config file for
+# the kernel. This file was generated by running `make defconfig`
+# followed by `make menuconfig` and then enabling the IPv6 protocol page.
+COPY disabled/aarch64-gnu/config /build/.config
+RUN curl https://cdn.kernel.org/pub/linux/kernel/v4.x/linux-4.4.42.tar.xz | \
+      tar xJf - && \
+      cd /build/linux-4.4.42 && \
+      cp /build/.config . && \
+      make -j$(nproc) all && \
+      cp arch/arm64/boot/Image /tmp && \
```

Comment:
I would yeah, but the normal build process didn't create that, so we didn't have it to use it :(